src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -353,6 +359,18 @@ private IEnumerable<FileInfo> FindTemplateFilesInDirectory(DirectoryInfo directo
             return armTemplates.Concat(bicepTemplates);
         }
 
+        private FileInfo FindParameterFileForTemplate(FileInfo template)
+        {
+            FileInfo parametersFile = null;
+            string defaultParametersFileLocation = Path.Combine(template.Directory.FullName, Path.GetFileNameWithoutExtension(template.Name) + ".parameters.json");
```

Comment:
[Here](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/parameter-files#file-name), its mentions that it can also be `azuredeploy.parameters-dev.json` with the `-dev` to identify the development environment. 
Thoughts on making it also work for this case? We can also add this as an issue for a future improvement.
[Method](https://learn.microsoft.com/en-us/dotnet/api/system.io.directory.getfiles?view=net-7.0#system-io-directory-getfiles(system-string-system-string)) and [example](https://stackoverflow.com/questions/1198721/directoryinfo-getfiles-beginning-with)

Reply:
Good one, hadn't thought of that 👍🏻 
Should now work, see [`b348f20` (#297)](https://github.com/Azure/template-analyzer/pull/297/commits/b348f205123045cb063639db6eed379ad5502301)